src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/player/ModuleFastUse.kt

```diff
@@ -114,11 +114,9 @@ object ModuleFastUse : Module("FastUse", Category.PLAYER) {
             if (player.activeItem.isFood || player.activeItem.item is MilkBucketItem
                 || player.activeItem.item is PotionItem
             ) {
-                if (player.isUsingItem) {
```

Comment:
I'm not sure about this to be honest, but I've meant paying attention to the line 97 =)